lib/zendrive/score.rb

```diff
@@ -32,7 +37,11 @@ def format_attrs(attributes)
       if attributes["info"] && attributes["info"].any? && attributes["info"]["distance_km"]
         attributes["info"]["distance_in_km"] = attributes["info"]["distance_km"].to_f
         attributes["info"]["distance_in_mi"] = attributes["info"]["distance_km"].to_f * 0.621371
-        attributes["info"]["drive_time_in_seconds"] = convert_drive_hours_to_seconds(attributes["info"]["drive_time_hours"])
+        if Zendrive.api_version == "v2"
+          attributes["info"]["drive_time_in_seconds"] = convert_drive_hours_to_seconds(attributes["info"]["drive_time_hours"])
+        elsif Zendrive.api_version == "v3"
+          attributes["info"]["drive_time_in_seconds"] = convert_drive_hours_to_seconds(attributes["info"]["duration_seconds"])
```

Comment:
Should this just be a straight assignment without going through the convert method?